metrics-operator/config/crd/bases/metrics.keptn.sh_analyses.yaml

```diff
@@ -18,6 +18,12 @@ spec:
     - jsonPath: .spec.analysisDefinition.name
```

Comment:
this file should be called differently, not with `_analyses` suffix. Also for `Analysis` CRD, we have plural defined as `analyses`. That might cause problems in the future

Reply:
which kinds of problems would you expect? I would keep it as is though, as this is what kubebuilder generates from the analysis go structs (since analyses is the plural form of analysis)

Reply:
ah right it was generated, ok we can leave it as it is